tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -18,7 +18,7 @@
 from typing import Optional, Any, Sequence, Tuple
 from tensornetwork.backends import base_backend
 from tensornetwork.backends.tensorflow import decompositions
-
+import tensorflow as tf
```

Comment:
Do not include this at the top. We don't want to assume everyone has tensorflow installed when we switch to having JAX be the default backend.

Reply:
Please still fix this